instructure/canvas-android
# [MBL-16470][Teacher] Integrate Inbox redesign

The changes mostly affect the Teacher app, but there were also some fixes and improvements for the core Inbox code as well:
- Animate avatars back when selection mode is exited.
- Collapsing filter header.

Test plan:
- Verify that all the previously working behavior works as intended (Showing Inbox list, paging, filtering by course/group, filtering by scope, opening messages, pull to refresh, writing messages, updating the list after writing a message or changing/deleting an opened message).
- Verify that the new bulk actions are working correctly.
- Verify that all the data shown is correct (check with the previous version).
- Verify that the routing works correctly.
- Smoke test Inbox in Student app.

refs: MBL-16470
affects: Teacher
release note: Added bulk actions to Inbox

## Checklist

- [x] Run E2E test suite or not needed
